### PR TITLE
Feature improve news

### PR DESCRIPTION
**Pull request by choi-hannah**
This PR closes issue #118 which was to remove explicit links to authors and issues. I have also added .Rhistory to the gitignore. Note: I've excluded one of them because it linked to a specific comment in the issue.

I have read the contributing guide and I have checked this contribution locally.

Note that I saw failures were running devtools::check but this was related to the stan model. Because my changes were unrelated to this, it's probably fine.

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # epinowcast 0.2.0
 
 ## Model
-- Added support for parametric log-logistic delay distributions. See [#128](https://github.com/epiforecasts/epinowcast/pull/128) by [@adrian-lison](https://github.com/adrian-lison).
+- Added support for parametric log-logistic delay distributions. See #128 by @adrian-lison.
```
Comment (seabbs) [inline NEWS.md:4]: Missing a news entry for this PR. Ideally, we need a comment saying what you have done and then linking to the PR and PR author (as done by Adrian here). 

As this is a package change can you add a ## Package subheading for your new entry as we did for `epinowcast 0.1.0`

**Review by seabbs — CHANGES_REQUESTED** [review]
Thanks for this @choi-hannah this looks great - thanks for the contribution! 

Just missing a news entry for this PR - so once that is added and CI checks are passing request a rereview and we should be okay to merge. 

Note that this would be a contributor-level addition for us so if you would like to be added to the DESCRIPTION as a `ctb` please do that in this PR.

File: NEWS.md (hunk 1 above)
Comment (choi-hannah) [reply NEWS.md:7]: I have done this

**Review by seabbs — APPROVED** [review]
Thanks looks good now and thanks for fixing the DESCRIPTION indenting!

Just waiting for the CI checks to complete and then will merge 🥳

**Pull request merged**